config/integration/apps.yml

```diff
@@ -60,20 +60,34 @@ dialogflow:
     }
   ]
   visible_properties: ['project_id']
-fullcontact:
```

Comment:
Removed the full contact integration option from the list as it is not working now.